src/app/stores/migrator.ts

```diff
@@ -636,4 +645,154 @@ export class Migrator {
 
     return state;
   }
+
+  public setMissedSortProperties(state: AppStoreState) {
+    for (const item of forEachObject(state.chart)) {
+      if (item.kind == ObjectItemKind.Chart) {
+        item.object.properties.exposed = true;
+      }
+      if (item.kind == ObjectItemKind.ChartElement) {
+        if (
+          Prototypes.isType(item.chartElement.classID, "plot-segment.cartesian")
+        ) {
+          const element = item.chartElement as PlotSegment<CartesianProperties>;
+          if (element.properties.xData) {
+            element.properties.xData = this.updateAxis(
+              element.properties.xData
+            );
+            if (element.properties.xData === undefined) {
+              element.properties.xData = null;
+            } else {
+              element.properties.xData.orderByCategories =
```

Comment:
Is this overwriting the property for `orderByCategories` even if there is a value for `orderByCategories`?

Reply:
Updated, thanks!